torchmetrics/text/bert.py

```diff
@@ -97,7 +97,7 @@ def __init__(
         self.all_layers = all_layers
         self.num_threads = num_threads
         self.batch_size = batch_size
-        self.device = device
+        self.embedding_device = device
```

Comment:
This is a bit counter-intutitive now. If I create this metric with `device=...` in params then it will not actually change the `metric.device`